### PR TITLE
feat(airflow): add astro-cli ingestion

### PR DESCRIPTION
**Pull request by Lee-W**
Closes: #190

### CODE REVIEW

**Comment by Lee-W** [comment]
圖片

**Review by sunank200 — COMMENTED** [review]
Also this change has to be tested as part of this issue: https://github.com/astronomer/ask-astro/issues/191

Requesting you to create new weaviate class for this change and deploy this change in dev before merging this.

File: airflow/include/tasks/extract/astro_cli_docs.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import re
+
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+from weaviate.util import generate_uuid5
+
+
+def extract_astro_cli_docs() -> list[pd.DataFrame]:
+    astronomer_base_url = "https://docs.astronomer.io"
```
Comment (sunank200) [inline airflow/include/tasks/extract/astro_cli_docs.py:12]: Can you add the doc strings please?

**Review by sunank200 — APPROVED** [review]

File: airflow/include/tasks/extract/astro_cli_docs.py (hunk 1 above)
Comment (Lee-W) [reply airflow/include/tasks/extract/astro_cli_docs.py:22]: done

**Review by pankajastro — APPROVED** [review]
LGTM

**Pull request merged**